docs/porting.md

```diff
@@ -153,6 +153,14 @@ after importing a module that does not support the GIL.
         endif
         ```
 
+    In CI, you will need to ensure a nightly cython is installed for
+    free-threaded builds and disable build isolation by passing
+    `--no-build-isolation` to `pip` at build time. If you
+    use `cibuildwheel` to produce wheels, you can conditionally set
+    `CIBW_BUILD_FRONTEND` to `pip; args: --no-build-isolation` for free-threaded
```

Comment:
This can also be `build; args: --no-build-isolation` if a project uses `build` for wheels.